client/ayon_core/hosts/houdini/plugins/create/create_dynamic.py

```diff
@@ -64,11 +130,30 @@ def create(self, product_name, instance_data, pre_create_data):
         if custom_instance_data:
             instance_data.update(custom_instance_data)
 
+        instance_data["families"] = ["dynamic"]
+
         # TODO: Add support for multiple representations
-        files = pre_create_data["files"]
-        representations = [create_representation_data(files)]
+        files = pre_create_data.get("files", [])
+        if files:
+            representations = [create_representation_data(files)]
+
+        output_paths = pre_create_data.get("output_paths", [])
+        if output_paths:
+            representations = []
+            for output_path in output_paths:
+                files = eval_files_from_output_path(
+                    output_path,
+                    instance_data["frameStart"],
+                    instance_data["frameEnd"])
```

Comment:
By doing this here - instead of requiring an explicit list of files as input to the creator basically means that this logic with resolving e.g. `$F4` which is essentially Houdini-specific now makes this Creator houdini targeted only whereas it would be great that this could work as an even lower-level generic creator ... anywhere.

So basically I do like this - but let's keep thinking about how, by design, this exact same lower level logic can run anywhere without being houdini specific. This "runtime instance" may very well run in a python process on the farm outside of Houdini for example.

Reply:
what about passing the representation to the creator. 
and in each DCC we should compute it and pass it to the creator. 
taking reference from fabia's demo. 
![image](https://github.com/BigRoy/ayon-core/assets/20871534/8199d95b-f2d1-4df0-b9d3-21a055b4fbc2)


and then creator can compute frame start and end from the input files.